shared_kv_storage/setup.py

```diff
@@ -68,15 +68,12 @@
     'lmdb'
 ]
 
-module_files = [
-    os.path.join(
-        tcf_root_dir, 'shared_kv_storage/kv_storage/remote_lmdb/db_store.i'),
+dbstore_csv_module = Extension(
```

Comment:
Can we add two extensions dbstore_module and dbstore_csv_module.
dbstore_module  - Basic DB store functions
dbstore_csv_module - CSV File format specific

`ext_modules` in `setup` can take list of Extensions` [dbstore_module, dbstore_csv_module]`